odo/core.py

```diff
@@ -169,10 +169,14 @@ def path(graph, source, target, excluded_edges=None, ooc_types=ooc_types):
                                     if issubclass(n, oocs)])
     with without_edges(graph, excluded_edges) as g:
         pth = nx.shortest_path(g, source=source, target=target, weight='cost')
-        edge = graph.edge
+        #https: // github.com / blaze / odo / issues / 579
+        #https://github.com/blaze/odo/issues/588
+        #node can be retrieved from graph via dict-like access
+        #edge = graph.edge
 
         def path_part(src, tgt):
-            node = edge[src][tgt]
+            #node = edge[src][tgt]
+            node = graph[src][tgt]
```

Comment:
Can you remove the comments? A git blame on github should suffice in referencing the change.

Reply:
It is done.